src/components/tab/SearchInput.vue

```diff
@@ -141,33 +157,38 @@ export default {
 				this.$emit('submit', link)
 				return
 			}
-			const params = new URLSearchParams()
-			params.append('workpackageId', selectedOption.id)
-			params.append('fileId', this.fileInfo.id)
-			params.append('fileName', this.fileInfo.name)
-			const config = {
-				headers: {
-					'Content-Type': 'application/x-www-form-urlencoded',
+			// since we can link multiple files now we send file information required in array (whether its only one value or multiple)
+			let fileInfoForBody = []
+			let successMessage
+			if (this.isSearchFrom === SEARCHFROM.PROJECT_TAB) {
+				fileInfoForBody.push(this.fileInfo)
+				successMessage = t('integration_openproject', 'Work package linked successfully!')
```

Comment:
I see in Figma the toast message is something like this, maybe we didn't follow through the message the previous time. Can we change it, this seems more accurate than what we are displaying right now. 
```suggestion
				successMessage = t('integration_openproject', 'Link to work package created successfully!')
```